spec/models/manageiq/providers/ibm_power_hmc/infra_manager/event_parser_spec.rb

```diff
@@ -0,0 +1,144 @@
+describe ManageIQ::Providers::IbmPowerHmc::InfraManager::EventParser do
+  class TestEvent
+    attr_accessor :type, :id, :published, :data, :detail, :usertask
+    def initialize
+      @type   = "ADD_URI"
+      @id     = "1639561179310"
+      @data   = "https://te.st:12443/rest/api/uom/ManagedSystem/977848c8-3bed-360a-c9d2-ae4b7e46b5d1"
+      @detail = "Other"
+    end
+  end
+
+  context "ManagedSystem" do
+    it "#event_to_hash" do
+      ems_id = "999"
+      event = TestEvent.new
+      expect(described_class.event_to_hash(event, ems_id)).to(
+        include(
+          :source       => 'IBM_POWER_HMC',
+          :event_type   => "ADD_URI",
+          :ems_ref      => "1639561179310",
+          :timestamp    => nil,
+          :ems_id       => ems_id,
+          :full_data    => {
+            :data     => "https://te.st:12443/rest/api/uom/ManagedSystem/977848c8-3bed-360a-c9d2-ae4b7e46b5d1",
+            :detail   => "Other",
+            :usertask => nil },
+          :host_ems_ref => "977848c8-3bed-360a-c9d2-ae4b7e46b5d1"
+        )
+      )
+    end
+  end
+
+  context "LogicalPartition" do
+    it "#event_to_hash" do
+      event = TestEvent.new
+      event.data = "https://te.st:12443/rest/api/uom/ManagedSystem/e4acf909-6d0b-3c03-b75a-4d8495e5fc49/LogicalPartition/74CC38E2-C6DD-4B03-A0C6-088F7882EF0E"
+      expect(described_class.event_to_hash(event, nil)).to(
+        include(
+          :vm_ems_ref  => "74CC38E2-C6DD-4B03-A0C6-088F7882EF0E",
```

Comment:
Surprised rubocop isn't warning about this, but the `=>` should be aligned with below

Reply:
Looks like the miq bot was slow to react on this new commit. But it eventually did.